### PR TITLE
[WIP] Update .env file to hide more sensitive information.

### PR DESCRIPTION
**Pull request by wlffann**
Add new variables to .env file to remove personal information from .yml files.

[#138940073]

### CODE REVIEW

**Comment by jasonnoble** [comment]
Please add to Contributing.md to explain how to find these new values.

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -40,8 +46,17 @@
...
+    ENV['PIVOTAL_PROJECT_ID']
+  end
+  c.filter_sensitive_data('<PIVOTAL_ACCOUNT_ID>') do |interation|
+    ENV['PIVOTAL_ACCOUNT_ID']
+  end
+  c.filter_sensitive_data('<PIVOTAL_ACCOUNT_NAME>') do |interaction|
+    ENV['PIVOTAL_ACCOUNT_NAME']
+  end
+  c.filter_sensitive_data('<PIVOTAL_MEMBERSHIP_ID>') do |interaction|
+    ENV['PIOVTAL_MEMBERSHIP_ID'].to_i
```
Comment (jasonnoble) [inline spec/spec_helper.rb]: Spelling...

**Comment by wlffann** [comment]
@jasonnoble I updated the Contributing.md file but I can't find the `PIVOTAL_MEMBERSHIP_ID` anywhere on Pivotal Tracker without making a request to the API. Since this was the variable that was giving trouble earlier, maybe it shouldn't be included in the .env file?

File: .env.example [hunk 2]
```diff
@@ -1,6 +1,10 @@
+FULL_NAME='YOUR_FULL_NAME'
```
Comment (jasonnoble) [inline .env.example:1]: Let's alphabetize these?

**Review by jasonnoble — CHANGES_REQUESTED** [review]

File: .env.example (hunk 2 above)
Comment (wlffann) [reply .env.example:1]: Good idea! I'm on it.

File: spec/spec_helper.rb (hunk 1 above)
Comment (wlffann) [reply spec/spec_helper.rb]: Thanks!

File: spec/lib/pivotal-tracker/account_spec.rb [hunk 3]
```diff
@@ -50,7 +50,7 @@
     end
 
     it "can add a new membership" do
-      membership_params = {:email => Faker::Internet.email, :name => Faker::Name.name, :initials => "AW"}
+      membership_params = {:email => 'laney.hoeger@senger.name', :name => 'Larue Mohr', :initials => 'LM'}
       new_membership = (account.memberships.create(membership_params))
```
Comment (lucyconklin) [inline spec/lib/pivotal-tracker/account_spec.rb:54]: This might be a good place for another factory once we get that branch merged in.

**Pull request closed without merging**